assets/rich_text_components_definitions.ts

```diff
@@ -78,7 +78,8 @@ export default {
       "schema": {
         "type": "unicode",
         "validators": [{
-          "id": "is_nonempty"
+          "id": "has_length_at_least",
+          "min_value": 5
```

Comment:
Do we need to ensure that all existing content in datastore passes this new validation?

Reply:
Good q -- Here's how we're thinking about all of these frontend validation changes: we won't make any guarantees about the datastore, but are simply doing some initial frontend work before we do audit jobs and corresponding backend validation changes.